src/main/java/nl/novi/behuiswerkles11/controllers/ExceptionsController.java

```diff
@@ -1,4 +1,15 @@
 package nl.novi.behuiswerkles11.controllers;
 
+import nl.novi.behuiswerkles11.exceptions.RecordNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
 public class ExceptionsController {
+    @ExceptionHandler (value = RecordNotFoundException.class)
+    public ResponseEntity<Object> exception(RecordNotFoundException exception) {
```

Comment:
De body van je response (de eerste parameter van de ResponseEntity constructor) is `exception.getMessage()`, dat is een String. De return type van deze methode mag dan ook `ResponeEntity<String>` zijn ipv `ResponseEntity<Object>` . Object wekrt ook, maar String is specifieker en daarm ee beter